pkg/api/customization/namespacedresource/namespaced_store.go

```diff
@@ -0,0 +1,39 @@
+package namespacedresource
```

Comment:
The file should be renamed to `store.go`. This is normal convention and go files usually don't have an underscore unless it's denoting the architecture the file is for i.e. `store_windows.go`